gcloud/pubsub/test__gax.py

```diff
@@ -376,3 +820,46 @@ class _ListTopicSubscriptionsResponsePB(object):
     def __init__(self, subscriptions, next_page_token=None):
         self.subscriptions = subscriptions
         self.next_page_token = next_page_token
+
+
+class _PushConfigPB(object):
+
+    def __init__(self, push_endpoint):
+        self.push_endpoint = push_endpoint
```

Comment:
These stubs are so lightweight / transparent that it just seemed right to let them go (I trashed one change which made them all use a common `DummyPB` class, for instance, because I liked the readability).